docs/notes/2.27.x.md

```diff
@@ -48,6 +48,10 @@ Minor fixes:
 
 - If a sandbox for executing mypy is preserved, the `__run.sh` script now refers to the main script by a relative path and [can thus be successfully executed](https://github.com/pantsbuild/pants/issues/22138).
 
+#### S3
+
+The S3 backend now uses Signature Version 4 for signing requests, allowing use of KMS encrypted objects in S3. The HMACV1 signing can be enabled by setting the `[s3-url-handler].auth_signing` option to `hmacv1`.
```

Comment:
Minor change: I think we can make it slightly easier for someone to confirm the behaviour of this option by anticipating the doc location and linking to it:

```suggestion
The S3 backend now uses Signature Version 4 for signing requests, allowing use of KMS encrypted objects in S3. The HMACV1 signing can be enabled by setting [the `[s3-url-handler].auth_signing` option](https://www.pantsbuild.org/2.27/reference/subsystems/s3-url-handler#auth_signing) to `hmacv1`.
```